runtime/Makefile

```diff
@@ -261,6 +242,15 @@ caml/jumptbl.h : caml/instruct.h
 caml/version.h : $(ROOTDIR)/tools/make-version-header.sh $(ROOTDIR)/VERSION
 	$^ > $@
 
+sak$(EXE): sak.$(O)
+	$(call MKEXE_USING_COMPILER,$@,$^)
+
+C_LITERAL = $(shell ./sak$(EXE) translate '$(1)')
+
+build_config.h: | sak$(EXE)
+	echo '#define OCAML_STDLIB_DIR $(call C_LITERAL,$(LIBDIR))' > $@
```

Comment:
Would it make sense to append a line of comment to explain that this file is generated ?

Reply:
Yes!